### PR TITLE
Test temporarily move away from pointer events

### PR DESCRIPTION
**Pull request by snowystinger**
Closes https://github.com/adobe/react-spectrum/issues/1338

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### CODE REVIEW

File: packages/@react-aria/menu/stories/useMenu.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,164 @@
+/*
```
Comment (snowystinger) [inline packages/@react-aria/menu/stories/useMenu.stories.tsx:1]: This is just a useful story for testing, i can move it into its own PR if we want

File: packages/@react-aria/interactions/stories/useInteractOutside.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
+/*
```
Comment (snowystinger) [inline packages/@react-aria/interactions/stories/useInteractOutside.stories.tsx:1]: This story demonstrates a bug we currently have, https://github.com/adobe/react-spectrum/issues/1338, that this PR does fix, though we need to evaluate if this is what we want to do.

File: packages/@react-aria/interactions/src/usePress.ts [hunk 3]
```diff
@@ -262,278 +272,295 @@ export function usePress(props: PressHookProps): PressResult {
       }
     };
 
-    if (typeof PointerEvent !== 'undefined') {
-      pressProps.onPointerDown = (e) => {
-        // Only handle left clicks
-        if (e.button !== 0) {
-          return;
-        }
+    // if (typeof PointerEvent !== 'undefined') {
```
Comment (ktabors) [inline packages/@react-aria/interactions/src/usePress.ts:275]: Can we remove the commented out code because it exists in the repo?

**Review by ktabors — COMMENTED** [review]
nothing jumped out at me when I read it and tested the new stories, but I think reading it while testing at a testing session would be better.

File: packages/@react-aria/interactions/src/usePress.ts (hunk 3 above)
Comment (snowystinger) [reply packages/@react-aria/interactions/src/usePress.ts:275]: Potentially, it does serve as a nice reminder, of what we're aiming for. But I don't have a strong opinion one way or the other

Comment (snowystinger) [reply packages/@react-aria/interactions/src/usePress.ts:275]: Talked to team, we're leaving them in as comment for now as a clear message to anyone looking at the code for why we aren't using pointerevents. Also letting it serve as a reminder to bring the code back.

**Pull request closed without merging**